src/less/toast/base/toast.less

```diff
@@ -0,0 +1,97 @@
+:root {
+    --ebay-skin-toast-spacing: 24px;
+}
+
+.toast__mobile {
+    --ebay-skin-toast-spacing: 16px;
+}
+
+.transition() {
+    transition:
+        opacity 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s,
+        transform 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s;
+}
+
+.toast {
+    background-color: @toast-box-color;
+    bottom: 16px;
+    color: @white-color;
+    left: 16px;
+    max-height: 40vh;
+    max-width: 480px;
+    min-width: 320px;
+    position: fixed;
+    z-index: 1;
+
+    &.toast--transition {
+        display: block;
+        transform: translateY(110%);
+    }
+}
+
+.toast.toast__mobile {
+    bottom: 0;
+    left: 0;
+    width: 100vw;
+}
+
+.toast.toast--transition.toast--hide {
+    opacity: 0;
+    transform: translateY(110%);
+
+    .transition();
+}
+
+.toast.toast--transition.toast--show {
+    opacity: 1;
+    transform: translateY(0);
+
+    .transition();
+}
+
+.toast__header {
+    align-items: center;
+    display: flex;
+    flex: 0 1 auto;
+    justify-content: space-between;
+    padding: var(--ebay-skin-toast-spacing);
+
+    > h3 {
+        margin: 0;
+        padding: 0;
+    }
+}
+
+.toast__close {
+    background-color: #3665f3;
+    border: 0;
+    color: #fff;
+    cursor: pointer;
+    margin: 0;
+}
+
+.toast > .toast__main {
+    margin: 0;
+    padding: var(--ebay-skin-toast-spacing);
+    padding-top: 0;
+}
+
+.toast__footer {
+    display: flex;
```

Comment:
You can use `.dialog-footer-content-large()` mixin for this. It should do the same. 

If you need additional styles, please add them to that as needed (and or change them to variables).
Or just add overrrides to the end